packages/turf-boolean-disjoint/index.js

```diff
@@ -0,0 +1,207 @@
+var inside = require('@turf/inside');
+var lineIntersect = require('@turf/line-intersect');
+var polyToLinestring = require('@turf/polygon-to-linestring');
+var invariant = require('@turf/invariant');
+var getGeom = invariant.getGeom;
+var getGeomType = invariant.getGeomType;
+var getCoords = invariant.getCoords;
+
+/**
+ * Boolean-disjoint returns (TRUE) if the intersection of the two geometries is an empty set.
+ *
+ * @name booleanDisjoint
+ * @param {Geometry|Feature<any>} feature1 GeoJSON Feature or Geometry
+ * @param {Geometry|Feature<any>} feature2 GeoJSON Feature or Geometry
+ * @returns {Boolean} true/false
+ * @example
+ * const point = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "Point",
+ *     "coordinates": [2, 2]
+ *   }
+ * }
+ * const line = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "LineString",
+ *     "coordinates": [[1, 1], [1, 2], [1, 3], [1, 4]]
+ *   }
+ * }
+ * turf.booleanDisjoint(line, point);
+ * //=true
+ */
+module.exports = function (feature1, feature2) {
+    var type1 = getGeomType(feature1);
+    var type2 = getGeomType(feature2);
+    var geom1 = getGeom(feature1);
+    var geom2 = getGeom(feature2);
+    var coords1 = getCoords(feature1);
+    var coords2 = getCoords(feature2);
+
+    switch (type1) {
+    case 'Point':
+        switch (type2) {
+        case 'Point':
+            return !compareCoords(coords1, coords2) || false;
```

Comment:
Not too sure if `|| false` is necessary?

If `compareCoords` only outputs `true/false` then if the result is `false` it would fallback again on `false`.

I'll look into this, but I'm pretty sure `|| false` can be dropped.